addons/l10n_il/data/fiscal_templates_data.xml

```diff
@@ -5,7 +5,6 @@
         <field name="name">Israel</field>
         <field name="chart_template_id" ref="il_chart_template"/>
         <field name="country_id" ref="base.il"/>
-        <field name="auto_apply" eval="True"/>
```

Comment:
Why?  If the partner needs another fiscal position, it can still be configured there and it can still be changed on the invoice.

Reply:
@jco-odoo 
What is the goal of an empty fiscal position which is auto apply like this one?

Reply:
@hedshefer To be sure it does not apply the others.

Reply:
@jco-odoo 
fixed